### ISSUE TITLE
Add version selection for TV Shows too

### ISSUE THREAD

**Issue by Sekki21956** [jellyfin-archive/jellyfin-roku-legacy#1276]
## Is your feature request related to a problem? Please describe
I have a few shows both in 4k and 1080p and I cannot select which one to play. It shows 4k in the overview but plays 1080p.

## Describe the solution you'd like
There is #575 which already allows the selection of audio tracks and the code already mentions the version selection could be added in [TVListOptions.brs](https://github.com/jellyfin/jellyfin-roku/blob/unstable/components/tvshows/TVListOptions.brs) I think adding an additional tab to that menu would be nice. This also already works for movies.

## Describe alternatives you've considered
Add an overview page for every episode like it is already done for movies where selection of versions is possible.

**Comment by jimdogx** [comment]
I was going to look in to this, however, I'm not able to get the different episodes to be recognized as one on the web client.  Are you able to do this on the web client?

My setup:

```
TVShows
    ShowName
          ShowName S01E01
               ShowName S01E01 - 2160p.mp4
               ShowName S01E01 - 480p.mp4
```

But they show up as too separate episodes on the web client (not one episode with different video versions).

**Comment by jimdogx** [comment]
Doing a little more research and I don't think the JF server currently supports this.  It looks like a feature that might come soon though:

https://github.com/jellyfin/jellyfin/pull/8004

**Comment by Sekki21956** [comment]
Yes this is not working automatically atm, but you can combine episode versions manually when you go to a details page and long press an episode in the bottom, select multiple episodes and group them.

**Comment by jimdogx** [comment]
What does your directory structure look like? I don't see a way to do it on the web so I tried the Android client... There is a group option but I can't get it to work.  Do I need to rename my episodes a certain way?


https://github.com/jellyfin/jellyfin-roku/assets/18198517/a697d035-51cf-48fa-b05b-19bc450c5761

**Comment by Sekki21956** [comment]
The last screen you were in, go to the overview of one of the two episodes. Then in the bottom there should be both of them. Long press one, tap on the other and now you should be able to group them via the three buttons in the top right corner "Versionen gruppieren" -> Group versions or somethitg like that.
![Screenshot_20230520-191937_Jellyfin](https://github.com/jellyfin/jellyfin-roku/assets/76734819/11b11e47-ad2e-431f-abfe-902cc9297f80)

**Comment by candry7731** [comment]
Should we close this?

**Comment by jimdogx** [comment]
Good call.  I thought it auto closed when the PR was merged, but apparently I didn't say "Fixes #1276", instead I just said "#1276".
